docs/index.md

```diff
@@ -1,68 +1,77 @@
 # Chisel
 
-**Chisel** is a tool for carving and cutting Ubuntu packages.
+**Chisel** is a developer tool for extracting well-defined portions ({{package_slices}}) of Ubuntu packages to create 
+compact, secure container images.
 
-It is built on the idea of {{package_slices}} - minimal, complementary, and
-loosely coupled sets of files, based on the package's metadata and content.
-Slices are basically logical subsets of the Ubuntu packages, with their own
-content and set of dependencies to other internal and external slices.
+Chisel extracts specialized slices and only installs the necessary files from Ubuntu packages.
 
-Chisel is able to extract a highly customised and specialised _Slice_ of the
-Ubuntu distribution, which one could see as a block of stone from which we can
-carve and extract the small and relevant parts we need to run our applications.
+Application creators need to be able to ship container images suited to their specific needs with a reduced attack 
```

Comment:
same comment as above -> we shouldn't make Chisel domain-specific. 

Chisel is for slices, what apt is for debs: a sw pkg manager. It can be any software (today just debs, but that's a technicality), and for any purpose (a container, a snap, etc.)